simplq/src/styles/joinPage.module.scss

```diff
@@ -13,3 +17,102 @@
     margin: 1rem;
   }
 }
+.header-title {
+  margin: auto;
+  .sub-header {
+    display: flex;
+    font-family: Pacifico;
+    font-size: 0.8rem;
+    font-weight: 300;
+    color: #3a3768;
+    align-items: center;
+  }
+}
+
+.header-bar {
+  @include background-color();
+  padding: 0.5rem;
+  // padding-top: 4rem;
+  
+  display: flex;
+  flex-wrap: wrap;
+  .simpleq-logo{
+    margin-top: 35px;
+  }
+  .header {
+    @include standard-header();
+    font-size: 2.7rem;
+    margin-bottom: 0;
+  }
+
+  .admin-button {
+    padding: 0 0.1rem;
+  }
+
+  .main-button-group {
+    @include center-horizontally();
```

Comment:
This is also not used anywhere it seems. Remove the unused classes.

Reply:
I just copied those css from main header component,  forget to remove it , will make clear in next PR. Thanks